src/style.css

```diff
@@ -86,10 +87,13 @@ nav ul li {
 
 main {
   padding: 0 2rem;
+  background-color: rgb(252, 238, 162);
 }
 
 img {
   width: 100%;
+  border-radius: 10px;
+  box-shadow: 1px 1px 1px 1px rgb(247, 252, 0);
 }
```

Comment:
The background color of Main and the img box shadow makes the UI design more color, So I suggest to make it none for accessibility purpose for interaction.